src/main/kotlin/uk/gov/dluhc/printapi/client/ElectoralRegistrationOfficeManagementApiException.kt

```diff
@@ -9,8 +9,8 @@ package uk.gov.dluhc.printapi.client
 
 abstract class ElectoralRegistrationOfficeManagementApiException(message: String) : RuntimeException(message)
 
-class ElectoralRegistrationOfficeNotFoundException(gssCode: String) :
-    ElectoralRegistrationOfficeManagementApiException("ERO for gssCode $gssCode not found")
+class ElectoralRegistrationOfficeNotFoundException(message: String) :
```

Comment:
@Neil-Massey , do you think it makes sense to change it to the below, so we guarantee all not found exception messages have the same format:

<pre>
class ElectoralRegistrationOfficeNotFoundException(searchCriteria: String) :
    ElectoralRegistrationOfficeManagementApiException("ERO not found for $searchCriteria")
    </pre>

Reply:
As above